api/proto/banyandb/database/v1/schema.proto

```diff
@@ -95,6 +96,8 @@ message FieldSpec {
   EncodingMethod encoding_method = 3 [(validate.rules).enum.defined_only = true];
   // compression_method indicates how to compress data during writing
   CompressionMethod compression_method = 4 [(validate.rules).enum.defined_only = true];
+  // aggregate_function indicates how to aggregate data
+  model.v1.AggregationFunction aggregate_function = 5;
```

Comment:
@hanahmily I think this is not the function I mentioned. This AggregationFunction is targeting for query aggregation, this API changes should be stream flowing process aggregation, right?

Reply:
Our plan is to use the same functions to handle these scenarios. These functions are pure computing logic without any context related to the processes they are in.